api/src/app.controller.ts

```diff
@@ -25,14 +26,55 @@ export class AppController {
     return this.appService.getGameState();
   }
 
-  @Get('marbleplayable')
-  isMarblePlayable(): Boolean {
-    // return this.appService.getIsMarblePlayable();
-    return false;
+  @Get('restartgame')
+  getRestartGame(): GameState {
+    return this.appService.getRestartGame();
+  }
+
+  @Post('gamestatehaschanged')
+  getGameStateHasChanged(@Body() body): boolean {
+    return this.appService.gameStateHasChanged(body);
+  }
+
+  @Post('marbleplayable')
+  getMarblePlayable(@Body() body): Boolean {
```

Comment:
Same. I know I told you `post` before but it seems I misunderstood the use case

Reply:
same as above